src/wp-includes/media.php

```diff
@@ -5662,7 +5662,7 @@ function wp_get_loading_optimization_attributes( $tag_name, $attr, $context ) {
 	}
 
 	// Special handling for programmatically created image tags.
-	if ( 'the_post_thumbnail' === $context || 'wp_get_attachment_image' === $context ) {
+	if ( 'the_post_thumbnail' === $context || 'wp_get_attachment_image' === $context || 'widget_media_image' === $context ) {
```

Comment:
Can you add this to be covered in tests as well?

Should be as simple to add it to the `data_special_contexts_for_the_content()` data provider method, except that this data provider is also used for a few older tests of the deprecated `wp_get_loading_attr_default()` function. Either we split the data provider method into an old and new one, or we remove the relevant old test coverage, since we will never modify that deprecated function in the future anyway. Keeping it is probably a bit safer, but either works IMO.

Reply:
@felixarntz Unit test added.